test/prop_uuid.erl

```diff
@@ -0,0 +1,34 @@
+-module(prop_uuid).
+-include_lib("proper/include/proper.hrl").
+
+prop_binary_codec() ->
+    ?FORALL(Val, proper_types:binary(16),
+            proper_lib:codec(pg_uuid, #{uuid_format => binary}, Val)).
+
+prop_string_codec() ->
+    ?FORALL(Val, proper_types:binary(16),
+            proper_lib:codec(
+              pg_uuid, #{uuid_format => string},
+              lists:flatten(strfmt(Val)),
+              fun canonic_str/1)).
+
+prop_bin_string_codec() ->
+    ?FORALL(Val, proper_types:binary(16),
+            proper_lib:codec(
+              pg_uuid, #{uuid_format => string},
+              iolist_to_binary(strfmt(Val)),
+              fun canonic_str/1)).
+
+prop_integer_codec() ->
+    ?FORALL(Val, proper_types:integer(
+                   -170141183460469231731687303715884105728,
+                   170141183460469231731687303715884105728),
```

Comment:
Should UUID be represented as signed integer?

Reply:
Yea, though pg_types supports integer, string and binary.

Reply:
I mean signed vs unsigned integer